main.rb

```diff
@@ -0,0 +1,61 @@
+require_relative 'app'
+
+def prompt
+  puts 'Welcome to the School Library App!'
+  loop do
+    display_options
+    option = take_input
+    option_methods(option)
+
+    break if option == 7
+  end
+end
+
+def display_options
+  puts 'Please choose one of the options: '
+  puts '1. - List all books'
+  puts '2. - List all people'
+  puts '3. - Create a person'
+  puts '4. - Create a book'
+  puts '5. - Create a rental'
+  puts '6. - List all rentals for a given person ID'
+  puts '7. - Exit'
+end
```

Comment:
- [ ] Dear @Reggeditt, You have done great with the project, I was able to add a `Book` successfully but fails to create a person. The app crash and throws `unknown keyword: :parent_permmission (ArgumentError)` error while creating a person. No matter a person's type it keeps throwing the error. Please make sure It's possible to create a person. 
- [ ] This also prevents me from creating a rent and checking the whole functionalities of the app. Please make sure the `rent` is working as expected once you make the earlier suggestion. 

### The error 
![image](https://github.com/Reggeditt/school-library/assets/69077061/32462d96-85df-4049-9499-260ed28d0de0)

Reply:
fixed